modules/policy-security-regulation-compliance.adoc

```diff
@@ -7,7 +7,7 @@
 = Security and regulation compliance
 
 
-Security and regulation compliance includes tasks such as the implementation of security controls  and compliance certification.
+Security and regulation compliance includes tasks such as the implementation of security controls and compliance certification.
```

Comment:
```suggestion
Security and regulation compliance includes tasks, such as the implementation of security controls and compliance certification.
```

Reply:
@skrthomas thank you for your review! This suggestion is now resolved.